src/serializers/PnmlSerializer.js

```diff
@@ -155,4 +159,51 @@ export default class PnmlSerializer {
         return labelElement;
     }
 
+    _getStyle (element) {
+        const style = { };
+        const rawStyle = element.metaObject.representation.attributes.style
+            .split(';').filter(Boolean);
+        for (let i=0; i<rawStyle.length; i++) {
+            const [ key, value ] = rawStyle[i].split(':');
+            style[key.trim()] = value.trim();
+        }
+        return style;
+    }
+
+    _createFillElement (doc, element) {
+        const style = this._getStyle(element);
+        let fill = { r: 112, g: 219, b: 147 };
```

Comment:
Why not just skip the fill element if no fill style is defined? That way the PNML is agnostic to the editor and will have that color in Renew anyways (as it is now).

Reply:
You're right. Sorry about this.